packages/payload/src/collections/config/types.ts

```diff
@@ -333,8 +341,7 @@ export type CollectionAdminOptions = {
   useAsTitle?: string
 }
 
-/** Manage all aspects of a data collection */
-export type CollectionConfig = {
+export type BaseCollectionConfig = {
```

Comment:
Any specific this rename was needed? I don't see any external references.

Reply:
This was used to redeclare them, but because that wasn't working, this is now reverted in https://github.com/payloadcms/payload/pull/4525